components/resources/ai_chat_ui_strings.grdp

```diff
@@ -72,7 +72,10 @@
   <message name="IDS_CHAT_UI_MODIFY_CONFIGURATION_LABEL" desc="A button label to configure custom models">
      Configure
   </message>
-  <message name="IDS_CHAT_UI_INTRO_MESSAGE_CHAT_BASIC" desc="AI Chat intro message for the default model">
+  <message name="IDS_CHAT_UI_INTRO_MESSAGE_CHAT_AUTOMATIC" desc="AI Chat intro message for the default model">
+    Model changes dynamically to use the best one for the task.<ph name="LINK_BEFORE">$1</ph>Learn more<ph name="LINK_AFTER">$2</ph>
```

Comment:
<img width="284" alt="Screenshot 2025-05-13 at 12 51 04 PM" src="https://github.com/user-attachments/assets/cb01f0b9-ac81-414f-a922-eca9f0d9b820" />
https://support.brave.com/hc/en-us/articles/26727364100493-What-are-the-differences-between-Leo-s-AI-Models (same as other models)